### ISSUE TITLE
nixpkgs some elements are not processed correctly

### ISSUE THREAD

**Issue by MOIS3Y** [catppuccin/gtk#74]
I use nixpkgs on Void Linux
This is a snippet from my home-manager:
```
  gtk = {
    enable = true;
    theme = {
      name = "Catppuccin-Mocha-Standard-Dark";
      package = pkgs.catppuccin-gtk.override {
        accents = [ 
          "blue" 
          #....
          ];
        variant = "mocha";
      };
    };
  }; 
```
The theme is selected from:
`echo $XDG_DATA_DIRS
/usr/local/share:/usr/share:/home/stepan/.nix-profile/share`
Everything works fine except for the control menu:
![Снимок экрана от 2023-01-31 14-15-10](https://user-images.githubusercontent.com/48302224/215672275-0840fe9d-a7d2-4e5c-9766-ad1cd0f0bd0d.png)

I've also tweaked the gtk4 apps manually so far:
 ```
ls -l ~/.config/gtk-4.0

assets -> /home/stepan/.nix-profile/share/themes/Catppuccin-Mocha-Standard-Blue-Dark/gtk-4.0/assets
gtk-dark.css -> /home/stepan/.nix-profile/share/themes/Catppuccin-Mocha-Standard-Blue-Dark/gtk-4.0/gtk-dark.css
gtk.css -> /home/stepan/.nix-profile/share/themes/Catppuccin-Mocha-Standard-Blue-Dark/gtk-4.0/gtk.css
settings.ini
 ```
I created the issue only because if you install from sources with a python script, the theme works like a charm without the mentioned problem and it's strange because the version in default.nix version = "0.4.1";
![Снимок экрана от 2023-01-30 18-35-28](https://user-images.githubusercontent.com/48302224/215673664-88f039c5-0bf1-42b4-a662-d6ae9a80aaa7.png)
GNOME Shell 43.2

I do not rule out that this is only my problem, so you can just close the issue.
In any case, thanks for your great work, I really appreciate it.

**Comment by rubyowo** [comment]
The nix pkg does build from source using the python script however

**Comment by MOIS3Y** [comment]
I found the problem, but haven't found a solution yet.
The problem seems to be file permissions and symbolic links in the gnome-shell directory.
```
tree ~/.nix-profile/share/themes/Catppuccin-Mocha-Standard-Blue-Dark/gnome-shell/
/home/stepan/.nix-profile/share/themes/Catppuccin-Mocha-Standard-Blue-Dark/gnome-shell/
├── assets
│   ├── calendar-arrow-left.svg
│   ├── calendar-arrow-right.svg
│   ├── calendar-today.svg
│   ├── checkbox-dark.svg
│   ├── checkbox-off-active.svg
│   ├── checkbox-off-hover.svg
│   ├── checkbox-off.svg
│   ├── checkbox.svg
│   ├── dash-placeholder.svg
│   ├── no-events.svg
│   ├── no-notifications.svg
│   ├── noise-texture.svg
│   ├── process-working.svg
│   ├── toggle-off.svg
│   ├── toggle-on-dark.svg
│   └── toggle-on.svg
├── gnome-shell.css
├── no-events.svg -> assets/no-events.svg
├── no-notifications.svg -> assets/no-notifications.svg
├── pad-osd.css
└── process-working.svg -> assets/process-working.svg
```
I just did an experiment, copied to ~/.local/share/themes/ theme from /nix/store/
Did a chown -R on my user but that didn't work.

As soon as I replaced the gnome-shell directory from the local copy of the theme that I installed from the git repo, the icons began to display normally.

I'll just leave this comment here in case anyone as crazy as me trying to use void and nix comes across this problem)
In fact, of course, it is worth switching completely to nixOS if you want to have a declarative configuration.

Thanks again for a great theme it's really awesome.

**Comment by MOIS3Y** [comment]
Hey guys hello!
I finally tested setting the theme declaratively on NixOS.
And the problem persists.
It is treated as I described above.
I think even so the problem is really in symbolic links inside the topic and how nix links it from / nix / store
![Снимок экрана от 2023-03-25 20-58-21](https://user-images.githubusercontent.com/48302224/227716373-04e7af59-b142-49c8-888c-75050a2cc4d9.png)
![Снимок экрана от 2023-03-25 23-33-47](https://user-images.githubusercontent.com/48302224/227723717-5b0f7b63-5765-4cf9-b4e0-55e35be95ca0.png)

**Comment by rubyowo** [comment]
interesting.

**Comment by vinicius507** [comment]
Similar problem here:

![image](https://user-images.githubusercontent.com/6608056/234179424-0d5d3589-b4c1-4bdd-923e-462312f7cd3e.png)

Also, gtk-4.0 symlinks should be added in NixOS?

**Comment by sgoudham** [comment]
Should be fixed via #151
